libdebug/debugger/internal_debugger.py

```diff
@@ -888,6 +888,23 @@ def finish(self: InternalDebugger, thread: ThreadContext, heuristic: str = "back
 
         self._join_and_check_status()
 
+    def _background_next(
+        self: InternalDebugger,
+        thread: ThreadContext,
+    ) -> None:
+        """Executes the next instruction of the process. If the instruction is a call, the debugger will continue until the function returns.
```

Comment:
see other comments

Reply:
Resolved in [fb11af7](https://github.com/libdebug/libdebug/pull/80/commits/fb11af7cf439a12391ae1831aee03fcd2b39b45f)